tests/test_operator_tools.py

```
@@ -258,6 +269,61 @@ def test_ccodex_dispatcher_is_installed_and_resolves_its_repo_root(self) -> None
                 subprocess.run(["bash", "-n", str(dispatcher)], capture_output=True).returncode, 0
             )
 
+    def test_ccodex_binds_the_pinned_ocx_and_jq_paths_at_install_time(self) -> None:
+        with tempfile.TemporaryDirectory() as temp:
+            root = Path(temp); config = self.config(root)
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)